### DISCUSSION TITLE
set nullable field as null

### DISCUSSION THREAD

**Discussion by ryanmerolle** [netbox-community/ansible_modules#426] [Q&A] [answered]
What is the method in which I can set a nullable field as null.  With mtu for example on an interrace, I pass null and I get an error that the var is a string instead of an int.

This could just be a gap in functionality.  If so, I can raise an issue.

Thanks!

**Comment by FragmentedPacket** [comment, marked as answer]
This is actually not possible at the moment.

There is #330 that is for this exact issue.

I'm contemplating the best way to provide this. The reason we remove null is to prevent people from having to specify everything, but looking into whether a state of `override` will work for the different use cases.
